### PR TITLE
New tests added for Task 9 (comments for review id). 

### PR DESCRIPTION
**Pull request by mgrindrod92**
Also added new code in controller, model, app and new utils controller

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -236,17 +238,69 @@ describe('GET/api/reviews', () => {
...
+});
+
+describe('GET//api/reviews/:review_id/comments', () => {
+    it('200: responds with an object (array) of comments for a given review', () => {
+        return request(app)
+            .get('/api/reviews/2/comments')
+            .expect(200)
+            .then(( body ) => {
+                console.log(body._body);
+                const reviewComments = body._body
```
Comment (elijaharley) [inline __tests__/app.test.js:262]: Make sure you remove any unnecessary console.logs Matt - are you able to destructure the body here?

File: controller/comments_controller.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        promises.push(selectCommentsByReviewId(review_id))
+    }
+    
+    Promise.all(promises)
+    .then(( [_, comments] ) => {
+// Output of first promise is nothing
+            res.status(200).send( comments )
+    })
+    .catch(next);
+}
```
Comment (elijaharley) [inline controller/comments_controller.js:19]: Nice implementation of Promise.all Matt. Make sure you take out any comments that aren't functional.

**Review by elijaharley — APPROVED** [review]
Greta stuff Matt - just take a look at removing extra console.logs and comments

File: __tests__/app.test.js (hunk 1 above)
Comment (mgrindrod92) [reply __tests__/app.test.js:262]: Done now. Cheers

**Pull request merged**